js/app/controllers/editorcontroller.js

```diff
@@ -59,6 +59,12 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 			{displayname: t('calendar', 'When shared show only busy'), type: 'CONFIDENTIAL'},
 			{displayname: t('calendar', 'When shared hide this event'), type: 'PRIVATE'}
 		];
+		
+		$scope.statusSelect = [
+		    {displayname: t('calendar', 'Tentative'), type: 'TENTATIVE'},
+   			{displayname: t('calendar', 'Confirmed'), type: 'CONFIRMED'},
```

Comment:
I feel like `CONFIRMED` should be the first option, because it's also the default. cc @jancborchardt